tripy/nvtripy/frontend/tensor.py

```diff
@@ -223,11 +235,26 @@ def eval(self) -> runtime.MemRefValue:
                 f"Note: Tensor was evaluated while compiling here: {str_from_stack_info(self.trace_tensor.eval_stack_info)}",
                 mode="once",
             )
+        return self
 
-        return data
+    def tolist(self) -> Union[List, numbers.Number]:
+        """
+        Returns the tensor as a nested list. If the tensor is a scalar, returns a python number.
```

Comment:
Does it make sense to also have an `item()` API for scalars? On one hand, it's redundant with `tolist`, but on the other it is more clear.